app/controllers/shipments_controller.rb

```diff
@@ -2,4 +2,15 @@ class ShipmentsController < ApplicationController
   def index
     @shipments = Shipment.all
   end
+
+  def show
+    @shipment = Shipment.find(params[:id])
```

Comment:
```suggestion
    @shipment = @company.shipments.find(params[:id])
```

Reply:
Thanks again for the advice, I actually noticed this while working on task 3 and changed it accordingly in my task 3 branch. Should I also update it on this branch?

Reply:
I would update in this PR if it belongs to the scope of this task. Mixing up two tasks would be hard to review in my opinion.